plugins/kubernetes/src/components/DeploymentsAccordions/DeploymentDrawer.tsx

```diff
@@ -66,6 +66,12 @@ export const DeploymentDrawer = ({
             Deployment
           </Typography>
         </Grid>
+        <Grid item>
+          <Chip
+            size="small"
+            label={`namespace: ${deployment.metadata?.namespace}` ?? 'default'}
```

Comment:
Would need to change should we need to introduce cluster scope resources, but I don't think we have any now.

Can you add a test?